test-cases/performance/perf-regression-latency-lwt-int.yaml

```diff
@@ -0,0 +1,29 @@
+test_duration: 300
+prepare_write_cmd: [
+  "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(update=1)' no-warmup cl=QUORUM n=2000000 -port jmx=6868 -mode cql3 native -rate threads=16"
+]
+
+stress_cmd_lwt_ine: "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(insert-if-not-exists=1)' no-warmup cl=QUORUM duration=10m -port jmx=6868 -mode cql3 native -rate threads=16"
+stress_cmd_lwt_uc: "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(update-if-cond=1)' no-warmup cl=QUORUM duration=10m -port jmx=6868 -mode cql3 native -rate threads=16"
+stress_cmd_lwt_ue: "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(update-if-exists=1)' no-warmup cl=QUORUM duration=10m -port jmx=6868 -mode cql3 native -rate threads=16"
+stress_cmd_lwt_de: "cassandra-stress user profile=/tmp/cs_lwt_perf_int.yaml 'ops(delete-if-exists=1)' no-warmup cl=QUORUM duration=10m -port jmx=6868 -mode cql3 native -rate threads=16"
+
+
+n_db_nodes: 3
+n_loaders: 3
+n_monitor_nodes: 1
+
+instance_type_loader: 'c4.2xlarge'
+instance_type_monitor: 't3.small'
+instance_type_db: 'i3.4xlarge'
+
+user_prefix: 'perf-regression-latency-lwt-int'
+space_node_threshold: 644245094
+ami_id_db_scylla_desc: 'VERSION_DESC'
+
+round_robin: true
+append_scylla_args: '--blocked-reactor-notify-ms 4 --experimental 1'
```

Comment:
maybe too small for LWT, check with @gleb-cloudius what is the bar we would like to see stalls from.

Reply:
Fixed